[
  {
    "poster": {
      "poster_name": "James Marx",
      "poster_handle": "james.marx"
    },
    "time_stamp": "2021-02-17 01:35:48",
    "text": "Hey there, I'm using the current get all Engagments API. I've used the get recent Engagments API before and it worked fine. My team is just cranking a lot more activity +10000 in the last 30 days. Obviously I'd need to use the get all api until v3 gets released. I've been getting such weird results trying to page through ie: offset=x.  I'd set the offset eg to 1 and get some results. Then I set it to like 1000 and get the same page of results? Am I missing something here? Thanks Guys.",
    "reply_count": 4,
    "reply_users_count": 3,
    "replies": [
      {
        "time_stamp": "2021-02-17 02:07:49"
      },
      {
        "time_stamp": "2021-02-17 07:27:18"
      },
      {
        "time_stamp": "2021-02-17 09:03:33"
      },
      {
        "time_stamp": "2021-02-18 07:05:21"
      }
    ]
  },
  {
    "poster": {
      "poster_name": "Kieran",
      "poster_handle": "kieran073"
    },
    "time_stamp": "2021-02-17 02:07:49",
    "edited": {
      "time_stamp": "2021-02-17 02:08:14"
    },
    "text": "Hey James - maybe a stupid question, but are you using the offset HubSpot responds with on the get all engagements api endpoint?"
  },
  {
    "poster": {
      "poster_name": "Fatima",
      "poster_handle": "fatimajaffal.1997"
    },
    "time_stamp": "2021-02-17 05:45:39",
    "text": "<@U38EZFVFE> ping !"
  },
  {
    "poster": {
      "poster_name": "Frederic Fauvel",
      "poster_handle": "ffauvel"
    },
    "time_stamp": "2021-02-17 07:27:18",
    "text": "<https://legacydocs.HubSpot.com/docs/methods/engagements/get-all-engagements|https://legacydocs.HubSpot.com/docs/methods/engagements/get-all-engagements>\n\nYou should start with offset at 0, then use the offset property return by your last call."
  },
  {
    "poster": {
      "poster_name": "Jamie Jackson",
      "poster_handle": "jamie340"
    },
    "time_stamp": "2021-02-17 08:35:18",
    "text": "<https://api.hubapi.com/api-catalog-public/v1/apis>\n\nis there not an engagements api?"
  },
  {
    "poster": {
      "poster_name": "Manosij Roy Chowdhury",
      "poster_handle": "manosij"
    },
    "time_stamp": "2021-02-17 08:35:19",
    "text": "I got what I was doing wrong, I must have passed the custom-property name in the properties array input of GET Contact By ID"
  },
  {
    "poster": {
      "poster_name": "James Marx",
      "poster_handle": "james.marx"
    },
    "time_stamp": "2021-02-17 09:03:33",
    "text": "Thanks guys, I missed that. So weird that it's random numbers."
  },
  {
    "poster": {
      "poster_name": "Jamie Jackson",
      "poster_handle": "jamie340"
    },
    "time_stamp": "2021-02-17 09:52:13",
    "text": "oh its not done yet...."
  },
  {
    "poster": {
      "poster_name": "Jamie Jackson",
      "poster_handle": "jamie340"
    },
    "time_stamp": "2021-02-17 09:52:29",
    "text": "is there a openapi defition for the old enagagements api?"
  },
  {
    "poster": {
      "poster_name": "Jamie Jackson",
      "poster_handle": "jamie340"
    },
    "time_stamp": "2021-02-17 10:00:13",
    "text": "is there an ETA on the new engagements api?"
  },
  {
    "poster": {
      "poster_name": "Sam Graim",
      "poster_handle": "sgraim"
    },
    "time_stamp": "2021-02-17 14:43:40",
    "edited": {
      "time_stamp": "2021-02-17 14:46:14"
    },
    "text": "Hello - I'm deleting records via the api using a python/flask application - during testing I'm intentionally trying to throw an error by just adding a bunch of characters onto a real HubSpot record ID, but for some reason HubSpot still accepts the delete request.\n\nMy code looks like:\n\n```def delete_sync_contact(r):\n    existing_contact = search_sync_contacts(r, [\"id\", \"email\", \"salesforce_account_id\", \"salesforce_contact_id\"], {\"propertyName\": \"salesforce_contact_id\", \"operator\": \"EQ\", \"value\": r['salesforce_contact_id']})\n\n    if existing_contact:\n        print(existing_contact)\n\n        url = \"<https://api.hubapi.com/crm/v3/objects/p7551458_sync_contact/{}>\".format(existing_contact['id'] + '1000000')\n        print(url)\n        querystring = {\"hapikey\":HAPIKEY}\n        headers = {\n            'accept': \"application/json\",\n            'content-type': \"application/json\"\n        }\n\n        response = requests.request(\"DELETE\", url, headers=headers, params=querystring)\n        print(response)\n        if response.status_code == 204:\n            return \"Success: sync contact deleted\", 204\n        else:\n            return response.text\n    else:\n        print(\"No contact found\")\n        return \"Error: no sync contact found\", 200```\nYou can see where I'm throwing in the `+ 1000000` in the url variable. Which is working and I can confirm that no record exists with that ID, however, HubSpot is still returning a 204, not an error message. The url prints out like this `<https://api.hubapi.com/crm/v3/objects/p7551458_sync_contact/761426771000000>` and HubSpot responds `&lt;Response [204]&gt;`"
  },
  {
    "poster": {
      "poster_name": "Chandanesh",
      "poster_handle": "chandanesh.kc"
    },
    "time_stamp": "2021-02-18 00:37:44",
    "text": "Hi All,\n \nGoal: Use HubSpot analytics for web traffic data.\n \nSo how to fetch/see the data for the HubSpot analytics? or is there any way to export the data of HubSpot analytics to SQL server.\n I have used Skyvia tool to export other data like companies, contacts, deals and etc.\nIs there any other tool to export the analytics data?\nSuggestion and help will be appreciated",
    "reply_count": 1,
    "reply_users_count": 1,
    "replies": [
      {
        "time_stamp": "2021-02-18 12:59:17"
      }
    ]
  }
]